src/lib/autoinstall/y2erb.rb

```diff
@@ -12,6 +12,13 @@ def self.render(path)
     class TemplateEnvironment
       include Yast::Logger
 
+      # @return [Boolean] whether the system was booted using UEFI or not according to linuxrc
+      def efi?
+        Yast.import "Linuxrc"
+
+        Yast::Linuxrc.InstallInf("EFI") == "1"
```

Comment:
in general looks good, I just add note that it needs to be in sycn with rules code as it should behave same

Reply:
I have moved the logic to a Module in order to share the code and also have adapted it in order to detect EFI boot in a normal Mode which is usefil for `ayast_probe`